README.md

```diff
@@ -80,6 +80,20 @@ $apiContext = ApiContext::restore($fileName);
 **Tip:** both saving and restoring the context can be done without any arguments. In this case the context will be saved
 to/restored from the `bunq.conf` file in the same folder with your script.
 
+##### Proxy
+
+You can use a proxy with the bunq PHP SDK. This option must be a string. This proxy will be used for all requests done with
+the SDK. You will be prompted to provide a proxy URL when using the interactive installation script.
```

Comment:
May be smth like `This proxy will be used for all the requests done with the context for which it was specified` ?